tardis/rest/app/security.py

```diff
@@ -1,127 +1,119 @@
+import secrets
 from ...configuration.configuration import Configuration
 from ...exceptions.tardisexceptions import TardisError
 
 from bcrypt import checkpw, gensalt, hashpw
-from fastapi import Depends, HTTPException, status
-from fastapi.security import OAuth2PasswordBearer, SecurityScopes
-from jose import JWTError, jwt
-from pydantic import BaseModel, ValidationError
+from fastapi import HTTPException, status, Depends
+
+from fastapi.security import SecurityScopes
+
+from pydantic import BaseModel
+from fastapi_jwt_auth import AuthJWT
 
-from datetime import datetime, timedelta
 from functools import lru_cache
-from typing import List, Optional
+from typing import Optional, List
```

Comment:
Why are you changing the order here? I would prefer alphabetically order for imports.

Reply:
I think this was auto-imports or my built in formatter again. Black or flage didn't complain